misc/core-derive/tests/test.rs

```diff
@@ -18,24 +18,23 @@
 // FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 // DEALINGS IN THE SOFTWARE.
 
+#![allow(dead_code)]
```

Comment:
I had purposefully avoided putting a single module-level attribute so that we don't ignore legitimate dead code warnings.

Reply:
I should've put this in a separate PR,